README.md

```diff
@@ -1,4 +1,6 @@
-![screenshot of a device detail page in Home Assistant](https://raw.githubusercontent.com/iMicknl/ha-tahoma/master/media/tahoma_device_page.png)
```

Comment:
Could you remove (or replace) the previous image? Not sure what is the best practice nowadays, but I hosted the image in this repo).

Reply:
Ah yes indeed, I don’t think this is required to push it within the repository.

Reply:
Done.